src/app/modules/webhook/webhook.service.ts

```diff
@@ -273,7 +273,7 @@ const getFormattedResponse = (
  */
 export const pushData = async (
   webhookUrl: WebhookParams['webhookUrl'],
-  submissionWebhookView: WebhookView,
+  submissionWebhookView: WebhookView | null,
```

Comment:
ideally the null check should be done by the calling function, but we can always refactor later. it's much easier to spot non-canonical function signatures when the types are explicitly declared.